salt/returners/pgjsonb.py

```diff
@@ -212,19 +225,18 @@ def _get_serv(ret=None, commit=False):
     try:
         # An empty ssl_options dictionary passed to MySQLdb.connect will
         # effectively connect w/o SSL.
-        ssl_options = {}
-        if _options.get('ssl_ca'):
-            ssl_options['ca'] = _options.get('ssl_ca')
-        if _options.get('ssl_cert'):
-            ssl_options['cert'] = _options.get('ssl_cert')
-        if _options.get('ssl_key'):
-            ssl_options['key'] = _options.get('ssl_key')
-        conn = psycopg2.connect(host=_options.get('host'),
-                                user=_options.get('user'),
-                                password=_options.get('pass'),
-                                database=_options.get('db'),
-                                port=_options.get('port'))
-#                                ssl=ssl_options)
+        ssl_options = {
+            k: v for k, v in _options.items()
```

Comment:
We should be using ``six.iteritems()`` here to ensure that Python 2 uses a generator. To do this, just add a ``from salt.ext import six`` to the imports, and change this line to the following:

```python
k: v for k, v in six.iteritems(_options)
```